File: openapi-test-java-server-quarkus-reactive/testapi/api/PetStatusChangedWebhook.java
package testapi.api;

import com.fasterxml.jackson.databind.JsonNode;
import io.smallrye.mutiny.Uni;
import java.lang.Void;

public interface PetStatusChangedWebhook {
  /** Called when a pet's status changes */
  Uni<Void> onPetStatusChanged(JsonNode body);
}

File: openapi-test-kotlin-server-quarkus-reactive/testapi/api/PetStatusChangedWebhook.kt
package testapi.api

import com.fasterxml.jackson.databind.JsonNode
import io.smallrye.mutiny.Uni
import java.lang.Void

interface PetStatusChangedWebhook {
  /** Called when a pet's status changes */
  fun onPetStatusChanged(body: JsonNode): Uni<Void>
}